gnovm/stdlibs/time/time.gno

```diff
@@ -1392,7 +1003,7 @@ func UnixMicro(usec int64) Time {
 
 // IsDST reports whether the time in the configured location is in Daylight Savings Time.
 func (t Time) IsDST() bool {
-	_, _, _, _, isDST := t.loc.lookup(t.Unix())
+	_, _, _, _, isDST := ChainTZ.lookup(t.Unix())
```

Comment:
Please add a test for this change.

Reply:
done: b45fc23
I'd like to know why a test is required for this particular function